### PR TITLE
Flaviaportinari caption

### PR DESCRIPTION
**Pull request by flaviaportinari**
Flavia slides

### CODE REVIEW

File: _posts/0000-01-07-flaviaportinari.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-![codercat](https://octodex.github.com/codercat.jpg)
+![codercat](https://octodex.github.com/femalecodertocat.jpg)
 {: .center}
```
Comment (flaviaportinari) [inline _posts/0000-01-07-flaviaportinari.md:8]: :)

**Review by flaviaportinari — COMMENTED** [review]
comment

**Review by AYMENCARLOS — APPROVED** [review]
ok

**Review by ghost — APPROVED** [review]

**Review by MorganeB — APPROVED** [review]
Approved

**Pull request merged**